bundles/plugins/org.bonitasoft.studio.common/src-test/java/org/bonitasoft/studio/common/jface/databinding/Status2MessageSeverityTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2015 Bonitasoft S.A.
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2.0 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.bonitasoft.studio.common.jface.databinding;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.eclipse.core.databinding.validation.ValidationStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.ui.forms.IMessage;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class Status2MessageSeverityTest {
```

Comment:
I would have put To instead 2 to stay consistent with other part of the Studio code but I don't have strong opinion on that